### PR TITLE
i18n(es): Update `customization.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- What does this PR change? Give us a brief description.
Updated `customization.mdx` based on these 2 commits 762c50165022f703552b555d013e87dc85314966 & c31b5570665f87aa136fbaa3d55ef361799c324a

### CODE REVIEW

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

File: docs/src/content/docs/es/guides/customization.mdx [hunk 1]
```diff
@@ -86,6 +86,19 @@ Puedes mostrar diferentes versiones de tu logo en modos claro y oscuro.
    }),
    ```
 
+## Habilitar el mapa del sitio
+
+Starlight tiene soporte incorporado para generar un mapa del sitio. Habilita la generación del mapa del sitio estableciendo tu URL como `site` en `astro.config.mjs`:
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/customization.mdx:91]:
```suggestion
Starlight tiene soporte incorporado para generar un sitemap. Habilita la generación del sitemap estableciendo tu URL como `site` en `astro.config.mjs`:
```

File: docs/src/content/docs/es/guides/customization.mdx [hunk 2]
```diff
@@ -86,6 +86,19 @@ Puedes mostrar diferentes versiones de tu logo en modos claro y oscuro.
    }),
    ```
 
+## Habilitar el mapa del sitio
```
Comment (kevinzunigacuellar) [inline docs/src/content/docs/es/guides/customization.mdx:89]:
```suggestion
## Habilitar el sitemap
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Thanks @dreyfus92, I wasn't sure if we should translate `sitemap` it appears that google keep it as is in their [docs](https://developers.google.com/search/docs/crawling-indexing/sitemaps/overview?hl=es).

File: docs/src/content/docs/es/guides/customization.mdx (hunk 1 above)
Comment (dreyfus92) [reply docs/src/content/docs/es/guides/customization.mdx:91]: Hello Kevin, I've found that there is a proper translation in spanish for `sitemap` in spanish https://support.microsoft.com/es-es/office/generar-un-mapa-de-sitio-web-329d4803-f99a-4332-a4db-7ac3fd0d97da, should I roll back?

File: docs/src/content/docs/es/guides/customization.mdx (hunk 2 above)
Comment (dreyfus92) [reply docs/src/content/docs/es/guides/customization.mdx:89]: Same here.

File: docs/src/content/docs/es/guides/customization.mdx (hunk 1 above)
Comment (kevinzunigacuellar) [reply docs/src/content/docs/es/guides/customization.mdx:91]: In that case all good ✅

**Comment by kevinzunigacuellar** [comment]
LGTM!

**Review by delucis — APPROVED** [review]
Thank you for the update @dreyfus92 and the reviews @Waxer59 and @kevinzunigacuellar! 💖

**Pull request merged**